src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/components/dot-edit-page-lock-info/dot-edit-page-lock-info.component.html

```diff
@@ -3,9 +3,9 @@
     class="page-info__locked-by-message"
     *ngIf="show"
 >
-    {{ dotMessageService.get('editpage.toolbar.page.locked.by.user', pageState.page.lockedByName) }}
+    {{ 'editpage.toolbar.page.locked.by.user' | dm:[pageState.page.lockedByName]}}
```

Comment:
what is the `:` for?

Reply:
is a parameter for the pipe. and it does the following, https://github.com/dotCMS/core-web/blob/master/src/app/api/services/dot-messages-service.ts#L118